scripts/jenkins/test-on-ci

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent none
+  environment {
+    BASE_DIR="src/github.com/elastic/hey-apm"
+    APM_SERVER_BASE_DIR = "src/github.com/elastic/apm-server"
+    JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
+    GO_VERSION = "${params.GO_VERSION}"
+    APM_SERVER_VERSION = "${params.APM_SERVER_VERSION}"
+  }
+  triggers {
+    cron('0 0 * * 1-5')
```

Comment:
can you run this a bit later, say 3am utc or something? the onld (so current) tests in infra run at a similar time, i think against the same elasticsearch cluster, right? so would be better that they don't interfere with each other.

Reply:
I have changed it to be executed on the first available time-slot starting at 3:00 AM UTC